qcschema/dev/dev_schema.py

```diff
@@ -35,11 +36,10 @@
                     "type": "string"
                 },
                 "basis": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Name of the basis set to be applied to the whole molecule"
                 },
-                "basis_spec": {
-                    "$ref": "#/definitions/basis_spec"
-                }
+                "basis_spec": basis.basis
```

Comment:
Should this just be `basis` and it can either be a string or a specification?

Reply:
That might be cleaner. I didn't look much into multiple types but if that can work then I have no complaints

Reply:
Sure, see [here](https://cswr.github.io/JsonSchema/spec/multiple_types/). I usually use the `anyOf` syntax.